### PR TITLE
Update Send a Payment golang example to use correct method

### PR DESCRIPTION
**Pull request by definitepotato**
**Summary**
For the example code of "Send a Payment" in the main() function lines 16 and 24 are not using the correct method, this should be client.AccountDetail() not client.Accounts()

**Explanation**
```
15:  destAccountRequest := horizonclient.AccountRequest{AccountID: destination}
16:  destinationAccount, err := client.Accounts(destAccountRequest)
17:  if err != nil { 
18:      panic(err)
19:  }
```

I'll use one of the two code blocks since they both have the same issue.  In the code block above line 15 creates an endpoint for a single account using the horizonclient.AccountRequest() method.  The following line 16 then sends a query to fetch accounts who have a signer.  Problem is this is an endpoint for multiple accounts, not a single account.  line 15 creates an endpoint for a single account, so line 16 is expecting horizonclient.AccountsRequest() not horizonclient.AccountRequest.

**Solution**
Change line 16 to call client.AccountDetail() rather than client.Accounts()

### CODE REVIEW

**Comment by tyvdh** [comment]
@rice2000 Who's on task to review golang api docs?

**Comment by rice2000** [comment]
Good question.  I'm not sure.  Is it out of reach for you @tyvdh?

**Comment by vcarl** [comment]
@Shaptic maybe? Last person at SDF to submit a Go example change

**Comment by Shaptic** [comment]
The Horizon team is prioritizing Protocol 14 testing right now, but yes, I can review Go-related SDK PRs once that's over.

@definitepotato Can you also update the partial snippets lower on [that page](https://developers.stellar.org/docs/tutorials/send-and-receive-payments/#send-a-payment)? This change currently only updates the large example, and they'd be out of sync.

**Comment by definitepotato** [comment]
@Shaptic absolutely, I'll go ahead and do that now.

**Review by Shaptic — APPROVED** [review]
@rice2000 @vcarl I don't have write access, but this LGTM. :+1: Thanks @definitepotato!

**Review by rice2000 — APPROVED** [review]
Fantastic!  Great work @definitepotato.  And thanks for looking it over, @Shaptic.  Team work. Dream work.

**Pull request merged**
